src/vehicle_use.cpp

```diff
@@ -995,7 +995,7 @@ void vehicle::operate_plow()
             g->m.ter_set( start_plow, t_dirtmound );
         } else {
             const int speed = velocity;
```

Comment:
consider `const int speed = abs( velocity );`

Reply:
I did, but could not decide where it would be best.

Reply:
Ideally, you'd compact both functions in to a single function with a lot of JSON trickiness, but I get why that's not what you want to do.

They're two separate functions, do it in each one.  Or leaving it how you have is fine, but since velocity is already an int, recasting it as a const int is a waste, while recasting it as abs() shows what you're doing.